.github/workflows/prbuild.yml

```diff
@@ -17,5 +19,13 @@ jobs:
           go-version: ${{ steps.info.outputs.go_version }}
       - run: make ci
       - run: make test-coverage
-      - name: upload coverage
+      - id: branch_info
+        uses: actions/github-script@v3
+        with:
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          script: |
+            const resp = await github.request('GET /repos/{owner}/{repo}/pulls/{pull_number}', {...context.repo, pull_number:context.payload.number});
+            core.setOutput('branch', resp.data.base.ref);
+      - if: ${{ steps.branch_info.outputs.branch == 'main' }}
```

Comment:
Why not upload coverage for all branches?

Reply:
Not sure if that's how it works.
Doesn't it just overwrite the coverage of the main branch?

Reply:
I don't think so. The ui seems to have branches and stuff. Even if not the action would take as input.

Reply:
done